delivery_package_type_number_parcels/models/stock_package_type.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class PackageType(models.Model):
+    _inherit = "stock.package.type"
+
+    number_of_parcels = fields.Integer("Number of parcels")
```

Comment:
@glitchov @jbaudoux Maybe something has to be more explained (in help here or in README) about what is really the number of parcels for a package type.